debugTestImage.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env tsx
+/**
+ * @file like runTestImages.ts, but for a single proposal,
+ * and leaves the chain running
+ */
+import { execSync } from 'node:child_process';
+import { parseArgs } from 'node:util';
+import { imageNameForProposal, readOneProposal } from './common';
+
+const options = {
+  match: { short: 'm', type: 'string' },
+} as const;
+const { values } = parseArgs({ options });
+
+const proposal = readOneProposal(values.match!);
+
+console.log(
+  // XXX Raph had a better way to get the container name than the jq below
```

Comment:
The idiom I've seen in automation is to use the container id printed by `docker run`.
Then I suppose you could `docker inspect` it to get the name.

Reply:
I found @raphdev 's comment https://github.com/Agoric/agoric-sdk/pull/8393/files#r1339194157